.github/workflows/publish-release-type.yaml

```diff
@@ -66,3 +66,21 @@ jobs:
           channel: team-pitch-black
           color: red
           verbose: true
+      - name: slack on success - aws-dev
+        if: ${{ success() }}
+        uses: actions-ecosystem/action-slack-notifier@fc778468d09c43a6f4d1b8cccaca59766656996a
+        with:
+          slack_token: ${{ secrets.slackToken }}
+          message: ":cool-doge: ${{ inputs.name }} successful"
+          channel: aws-dev
+          color: green
+          verbose: false
+      - name: slack on failure - aws-dev
+        if: ${{ failure() }}
```

Comment:
Yeah this should be okay:

>success
>Returns true when none of the previous steps have failed or been canceled.

So none of them. If any failed, this will just fail. Should be okay. :)